tasks/gulp/backstop-config.js

```diff
@@ -81,17 +81,14 @@ module.exports = ({ host, port, components }) => ({
   engine: 'puppeteer',
   engineOptions: {
     args: ['--no-sandbox'],
-    // executablePath: '/usr/bin/chromium', // workaround for Apple Silicon
   },
+  // if running locally, and either stage hangs, try reducing these limits
   asyncCaptureLimit: 5,
   asyncCompareLimit: 50,
   debug: false,
   debugWindow: false,
   misMatchThreshold: 0,
   dockerCommandTemplate: 'docker run --rm -it --network host --mount type=bind,source="{cwd}",target=/src backstopjs/backstopjs:{version} {backstopCommand} {args}',
-  // workaround for Apple Silicon
-  // dockerCommandTemplate: 'docker run --rm -it --network host --mount type=bind,source="{cwd}",
-  // target=/src dockerman33/backstopjs:5.4.4 {backstopCommand} {args}',
```

Comment:
This was a temporary workaround I documented last year when I was unable to run the tests on my M1 machine.  But backstopjs has since been bumped to v6 and now works out of the box.